executor/executor_bsd.cc

```diff
@@ -21,12 +21,24 @@
 #define __syscall syscall
 #endif
 
+#include <fcntl.h>
 #include <signal.h>
+#include <sys/ioctl.h>
 #include <sys/mman.h>
 #include <sys/resource.h>
 #include <sys/time.h>
 #include <sys/types.h>
```

Comment:
In general `<sys/types.h>` should be first.

Reply:
Is it documented anywhere? Header files generally must be self contained. I noticed that sys/types.h usually goes first in man pages, but it's unclear if it's actually a requirement or different headers pull in different things and they just happen to be in this order.
We use clang-format which sorts includes. So if we need to preserve a particular order we would need to separate all of them with empty lines, which is somewhat ugly.
I will merge this now (it's a good time for syzbot), but we can always fix this later if something breaks.